src/pwa-buildpack/project-setup/create-configuration-files/index.md

```diff
@@ -19,10 +20,24 @@ Create a `.babelrc` file in your theme’s root directory with the following con
 }
 ```
 
-This Babel configuration file allows you to use [JSX] in your project.
-JSX allows you to write React components using a syntax structure similar to HTML.
-
-See [Introducing JSX] for more information.
+This configuration installs the following [Babel] plugins:
+
+* [`syntax-jsx`] and [`transform-react-jsx`] - Transform [JSX] code into backwards-compatible JavaScript. 
+  JSX allows you to write React components using a syntax structure similar to HTML.
+    
+  See [Introducing JSX] for more information.
+* [`transform-class-properties`] - Transforms class properties and lets you use the following syntax:
```

Comment:
It may not be sustainable for you to document each Babel plugin in detail the way you have, but for now, I appreciate it.